angular/base/src/polyfills.ts

```diff
@@ -42,8 +42,6 @@
  *
  */
 
-import './zone-flags';
```

Comment:
We should continue to import this file into the polyfills file. I accidentally removed it during another updated and Mike re-added it with the standalone component work, but it isn't directly associated to standalone components.